src/Stratis.Bitcoin.Networks/StratisMain.cs

```diff
@@ -84,9 +84,17 @@ public StratisMain()
 
             var bip9Deployments = new StratisBIP9Deployments()
             {
-                [StratisBIP9Deployments.ColdStaking] = new BIP9DeploymentsParameters(2,
+                [StratisBIP9Deployments.ColdStaking] = new BIP9DeploymentsParameters(2, 
                     new DateTime(2018, 12, 1, 0, 0, 0, DateTimeKind.Utc),
-                    new DateTime(2019, 12, 1, 0, 0, 0, DateTimeKind.Utc))
+                    new DateTime(2019, 12, 1, 0, 0, 0, DateTimeKind.Utc)),
+
+                [StratisBIP9Deployments.CSV] = new BIP9DeploymentsParameters(3, 
```

Comment:
Instead of bit 3, shouldn't this be 1?

```suggestion
                [StratisBIP9Deployments.CSV] = new BIP9DeploymentsParameters(1, 
```

Reply:
hmm I guess it could we should have kept a slot for CSV and segwit in advance instead of cold staking 😞 too late now.

Reply:
Yes 😢